src/storage/storage.cc

```diff
@@ -56,6 +56,23 @@ const int64_t kIORateLimitMaxMb = 1024000;
 
 using rocksdb::Slice;
 
+const static std::map<std::string, std::string> compression_sort_long_string_map = {{"no", "kNoCompression"},
```

Comment:
I think the following map is sufficient. Why do we need `compression_sort_long_string_map`?

Reply:
setOptions requires parameters of type string